sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_response_handlers.py

```diff
@@ -59,8 +59,31 @@ def order_results(response, combined):
 
 
 def prepare_result(func):
+    def _deal_with_too_many_documents(response, obj):
+        # special case for now if there are too many documents in the request
+        # they may change id to empty string, but currently it is "All"
+        too_many_documents_errors = [
+            error for error in obj.errors
+            if error.id == "All" and error.error.innererror.code == "InvalidDocumentBatch"
```

Comment:
we're accessing innererror here, I think we need to move that hasattr check above this